UKHO.PeriodicOutputService/UKHO.BESS.API.FunctionalTests/FunctionalTests/ScsIntegration/ScsIntegrationTests.cs

```diff
@@ -20,23 +20,15 @@ public async Task OneTimeSetupAsync()
         [Test]
         public async Task WhenICallScsEssDataEndpointWithValidToken_ThenSuccessStatusCode200IsReturned()
         {
-            HttpResponseMessage apiResponse = await ScsEndpointHelper.ScsEssDataEndpoint(testConfiguration.scsConfig.BaseUrl, ScsJwtToken);
+            HttpResponseMessage apiResponse = await ScsEndpointHelper.ScsEssDataEndpoint(testConfiguration.scsConfig.MockBaseUrl, ScsJwtToken);
             apiResponse.StatusCode.Should().Be((HttpStatusCode)200);
         }
 
-        //PBI 140034: BESS CS - Get AVCS catalogue from SCS
-        [Test]
-        public async Task WhenICallScsEssDataEndpointWithInValidToken_ThenUnauthorizedStatusCode401IsReturned()
-        {
-            HttpResponseMessage apiResponse = await ScsEndpointHelper.ScsEssDataEndpoint(testConfiguration.scsConfig.BaseUrl, testConfiguration.authTokenConfig.FakeToken);
-            apiResponse.StatusCode.Should().Be((HttpStatusCode)401);
-        }
-
         //PBI 140034: BESS CS - Get AVCS catalogue from SCS
         [Test]
         public async Task WhenICallInvalidScsEssDataEndpointWithValidTokenAndWithIncorrectURL_ThenBadRequestStatusCode400IsReturned()
```

Comment:
Change the test name to WhenICallEssDataEndpointWithValidTokenAndIncorrectURL_ThenBadRequestStatusCode400IsReturned